test/e2e/nodelatencymonitor_test.go

```diff
@@ -0,0 +1,95 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	"antrea.io/antrea/pkg/features"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func TestNodeLatencyMonitor(t *testing.T) {
+	skipIfFeatureDisabled(t, features.NodeLatencyMonitor, true, false)
+	skipIfHasWindowsNodes(t) // as the feature does not have official windows support
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Error when setting up test: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	//  1: create the NodeLatencyMonitor CR with a 10s ping interval
+	_, err = data.crdClient.CrdV1alpha1().NodeLatencyMonitors().Create(context.TODO(), &crdv1alpha1.NodeLatencyMonitor{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "default",
+		},
+		Spec: crdv1alpha1.NodeLatencyMonitorSpec{
+			PingIntervalSeconds: int32(10),
+		},
+	}, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("Failed to create NodeLatencyMonitor CR: %v", err)
+	}
+	t.Logf("NodeLatencyMonitor CR created successfully.")
+
+	//  2: Poll until(5min) NodeLatencyStats are reported correctly
+	ctx := context.Background()
```

Comment:
why use `context.Background()` here when you use `context.TODO()` for other API calls?